proto/semantic.proto

```diff
@@ -161,12 +162,24 @@ enum NodeType {
   REFERENCE = 4;
 }
 
+enum SyntaxType {
+  UNKNOWN = 0;
+  FUNCTION = 1;
+  METHOD = 2;
+  CLASS = 3;
+  MODULE = 4;
+  CALL = 5;
+  TYPE = 6;
+  INTERFACE = 7;
+  IMPLEMENTATION = 8;
+}
+
 message StackGraphNode {
   int64 id = 1;
   string name = 2;
   string line = 3;
-  string kind = 4;
-  Span span = 5;
+  Span span = 4;
+  SyntaxType syntax_type = 5;
```

Comment:
Is it ok to remove `kind` and renumber the fields in the protobuf? Will we need to do some kind of simultaneous rollout of the change on the client and the server?

Reply:
It's only OK b/c we don't use this endpoint at all yet.